server_code/historic_server/__init__.py

```diff
@@ -47,27 +60,64 @@ def save_events(
     return identifiers if return_identifiers else None
 
 
-@anvil.server.callable
-def save(obj, projectors=None):
-    event_type = "creation" if obj.uid is None else "change"
-    return_identifiers = True if event_type == "change" else False
-    event = Event(event_type, obj)
-    identifier = save_event_records(event, return_identifiers=return_identifiers)[0]
-    play_projectors(projectors)
-    return identifier
+@anvil.server.callable("anvil_labs.historic.create")
+def create(obj, projectors=None):
```

Comment:
For these server callables

I also meant that the client side historic module should have a `create()` function - to make the public api nicer
And then these functions could be imported on both server and client.
The public facing api would just be a wrapper for doing `anvil.server.call`

```python
def create(obj, projectors=None):
    """"nice doc string for anvil's autocompleter"""
     return anvil.server.call("anvil_labs.historic.create", obj, projectors=projectors)
```